sdk/python/plexi_sdk/_render_context.py

```
@@ -500,6 +500,52 @@ def list_view(self, items: "list[dict]", selected: int = 0,
                      "items": items, "selected": selected,
                      "item_height": item_height})
 
+    def list_view(
+        self,
+        id: str,
```

Ruff (0.15.14)

[error] Function argument `id` is shadowing a Python builtin

(A002)